### ISSUE TITLE
App crashes upon Todoist sync

### ISSUE THREAD

**Issue by tejasraman** [flathub/com.github.alainm23.planner#12]
If I log in with Todoist, the app crashes.

Fedora KDE 36
Kernel 5.19.12

**Comment by hrqmonteiro** [comment]
It doesn't only crashes. It literally doesn't work with Todoist.

When it doesn't crashes, you add the Tasks and it disappears. It is completely broken.

**Comment by tejasraman** [comment]
True... It worked before the update to planner. I tried the old version and it worked.

**Comment by hrqmonteiro** [comment]
> True... It worked before the update to planner. I tried the old version and it worked.

How did you get the old version, please? I could try to use it for now as well.

**Comment by tejasraman** [comment]
See this: https://github.com/flatpak/flatpak/issues/3097
